### PR TITLE
12.0 -> 13.0 (manual)

### PR DESCRIPTION
**Pull request by em230418**

### CODE REVIEW

File: website_multi_company_portal/__manifest__.py [hunk 1]
```diff
@@ -32,5 +32,5 @@
     "uninstall_hook": None,
 
     "auto_install": False,
-    "installable": False,
+    "installable": True,
```
Comment (yelizariev) [inline website_multi_company_portal/__manifest__.py]: Is it tested?

File: website_sale_no_sale/__manifest__.py [hunk 2]
```diff
@@ -29,5 +30,5 @@
     "uninstall_hook": None,
 
     "auto_install": False,
-    "installable": False,
+    "installable": True,
```
Comment (yelizariev) [inline website_sale_no_sale/__manifest__.py]: is it tested in 13?

File: website_multi_company_portal/__manifest__.py (hunk 1 above)
Comment (em92) [reply website_multi_company_portal/__manifest__.py]: Taken away https://github.com/it-projects-llc/website-addons/pull/495/commits/012f301a31831fb1b77b8da7dc2a9d1b6c37f196

File: website_sale_no_sale/__manifest__.py (hunk 2 above)
Comment (em92) [reply website_sale_no_sale/__manifest__.py]: Same. Marked as non installable. https://github.com/it-projects-llc/website-addons/pull/495/commits/5ed8ac43b36e92b023833bd6f38c8778e628f133

**Comment by em230418** [comment]
I approve to merge it now

**Pull request merged**